aurora/src/test/java/com/market/banica/aurora/mapper/RequestMapperTest.java

```diff
@@ -40,15 +40,38 @@
 @ExtendWith(MockitoExtension.class)
 class RequestMapperTest {
 
-    private static final Aurora.AuroraRequest AURORA_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("aurora/eggs/10").build();
-    private static final Aurora.AuroraRequest ORDERBOOK_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("orderbook/eggs/10").build();
-    private static final Aurora.AuroraRequest ORDERBOOK_SUBSCRIBE_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("orderbook/eggs=subscribe").build();
-    private static final Aurora.AuroraRequest ORDERBOOK_UNSUBSCRIBE_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("orderbook/eggs=unsubscribe").build();
-    public static final Aurora.AuroraRequest MARKET_AVAILABILITY_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("market-europe/availability/eggs/2.50/2").build();
-    public static final Aurora.AuroraRequest MARKET_RETURN_PENDING_PRODUCT_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("market-europe/return/eggs/2.50/2").build();
-    public static final Aurora.AuroraRequest MARKET_BUY_PRODUCT_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("market-europe/buy/eggs/2.50/2").build();
-
-    private static final Aurora.AuroraRequest INVALID_REQUEST = Aurora.AuroraRequest.newBuilder().setTopic("market/banica").build();
+    public static final Aurora.AuroraRequest MARKET_AVAILABILITY_REQUEST = Aurora.AuroraRequest
```

Comment:
these don't seem to exceed the 1 screen rule, so they were good in the initial state